src/redux/reducers/exchange/reducer.js

```diff
@@ -10,24 +11,43 @@ export default (state = initialState, action) => {
   switch (action.type) {
     case EXCHANGE_MAKE_INIT: {
       return {
-        ...state,
-        isLoading: true,
-        isError: false,
-        error: initialState.error
+        transactionRecords: [],
+        ...state
       };
     }
     case EXCHANGE_MAKE_SUCCESS: {
       return {
         ...state,
-        isLoading: false
+        isLoading: false,
+        transactionRecords: [
+          ...state.transactionRecords,
+          {
+            time: DateTime.now().toISO(),
```

Comment:
Reducer should only take data and insert them into structure. Time should be set up on `makeExchange()` level.